script.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
```

Comment:
Cold you explain why strict mode is required here? @mujeeb100

Reply:
code should be executed in the strict mode,
if you are not declaring the variable in the function it allows you to error
mistyping a variable name creates a new global variable and all others things,
syntax error